.github/CODEOWNERS

```diff
@@ -2,61 +2,37 @@
 /app/models/container*                          @agrare
-/app/models/conversion_host*                    @fdupont-redhat
-/app/models/dialog*                             @tinaafitz
 /app/models/ems_event*                          @agrare
 /app/models/ems_refresh*                        @agrare
-/app/models/event_stream*                       @agrare @lfu
-/app/models/file_depot*                         @NickLaMuro @jerryk55
+/app/models/event_stream*                       @agrare
+/app/models/file_depot*                         @NickLaMuro
 /app/models/firmware_registry*                  @agrare
-/app/models/generic_object_definition*          @lfu
 /app/models/job*                                @agrare
 /app/models/manageiq/providers                  @agrare
 /app/models/metric*                             @agrare
 /app/models/miq_cockpit_ws_worker*              @jrafanie
-/app/models/miq_dialog*                         @tinaafitz
 /app/models/miq_ems_metrics_processor_worker*   @agrare
 /app/models/miq_event_handler*                  @agrare
-/app/models/miq_policy*                         @lfu
-/app/models/miq_product_feature*                @h-kataria
-/app/models/miq_provision*                      @gmcullough
-/app/models/miq_remote_console_worker*          @skateman
-/app/models/miq_report*                         @kbrock @lpichler
-/app/models/miq_request*                        @gmcullough
+/app/models/miq_report*                         @kbrock
 /app/models/miq_schedule*                       @jrafanie
-/app/models/miq_search*                         @lpichler
 /app/models/miq_server*                         @jrafanie
 /app/models/miq_template*                       @agrare
-/app/models/miq_widget*                         @yrudman
 /app/models/miq_worker*                         @jrafanie
-/app/models/notification*                       @skateman
-/app/models/orchestration_stack*                @lfu
-/app/models/orchestration_template*             @lfu
 /app/models/physical_server*                    @agrare
-/app/models/policy_event*                       @lfu
 /app/models/scan_item*                          @agrare
-/app/models/service*                            @gmcullough
-/app/models/transformation_mapping*             @fdupont-redhat
 /app/models/vim_performance_state*              @agrare
 /app/models/vm                                  @agrare
-/app/models/vm_migrate_workflow                 @fdupont-redhat
 /app/models/vm_or_template                      @agrare
 /bin                                            @jrafanie @bdunne
 /content                                        @bdunne
 /config                                         @jrafanie
 /db                                             @agrare @bdunne
 /lib/generators                                 @agrare
 /lib/manageiq/reporting                         @jrafanie
-/product*                                       @h-kataria
```

Comment:
Maybe we should replace @h-kataria with @kavyanekkalapu as she is now our main UI engineer